release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -82,6 +82,10 @@ In {product-title} {product-version}, you can install a cluster that uses a NAT
 ==== Installing a cluster on Google Cloud Platform (GCP) using pd-balanced disk type
 In {product-title} {product-version}, you can install a cluster on GCP using the `pd-balanced` disk type. This disk type is only available for compute nodes and cannot be used for control plane nodes. For more information, see xref:../installing/installing_gcp/installation-config-parameters-gcp.adoc#installation-configuration-parameters-additional-gcp_installation-config-parameters-gcp[Additional GCP configuration parameters].
 
+[id="ocp-4-14-optional-capabilities"]
+==== Optional capabilities in {product-title} {product-version}
+In {product-title} {product-version}, you can disable the `Build`, `DeploymentConfig`, `ImageRegistry` and `MachineAPI` capabilities during installation. You can only disable the `MachineAPI` capability if you install a cluster with user-provisioned infrastructure. For more information, see xref:../installing/cluster-capabilities.adoc#cluster-capabilities[Cluster capabilities].
```

Comment:
Same comment as your other PR about using "4.14" instead of "{product-version}" here

Also, I did already add a section on disabling the Build and DeploymentConfig capabilities, here: https://docs.openshift.com/container-platform/4.14/release_notes/ocp-4-14-release-notes.html#ocp-4-14-dc-build-apis-disabled

Not sure how you want to handle this, if you want to remove mine in favor of this one?

Reply:
Jeez, thanks for reminding me about that. I will remove yours in this PR so that all of the capabilities are mentioned in the same note.